front/components/data_source_view/DataSourceNodeTable.tsx

```diff
@@ -277,26 +203,22 @@ export function DataSourceNodeTable({
     ]
   );
 
-  const isLoading =
-    !traversedNode && selectedCategory ? isCategoryLoading : isChildrenLoading;
-
   return (
     <div>
```

Comment:
`nit`: Not related to this PR but we probably wanna add a `h-full` or something here?